docs/README.md

```
@@ -1,46 +1,69 @@
 # Lilia Documentation
 
+
 This repository contains the source files for the official Lilia Wiki documentation.
 
-> **Note:**  
-> The documentation lives in the `docs/` directory of the main Lilia repo:  
-> https://github.com/LiliaFramework/Lilia/tree/main/docs  
+
+> **Note:**
+
```

markdownlint-cli2 (0.17.2)

Blank line inside blockquote
null

(MD028, no-blanks-blockquote)